src/riak_core_bucket.erl

```diff
@@ -24,17 +24,18 @@
 -module(riak_core_bucket).
 
 -export([append_bucket_defaults/1,
-    set_bucket/2,
```

Comment:
While we're at it, what do you think of making them one-per line and sorting? Take all the diff-pain in one go.